File: Dagger-Android-Sample/src/main/java/com/gabrielsamojlo/dagger_212/MainActivity.java
package com.gabrielsamojlo.dagger_212;

import android.os.Bundle;
import android.widget.TextView;

import javax.inject.Inject;

import dagger.android.support.DaggerAppCompatActivity;

public class MainActivity extends DaggerAppCompatActivity {

    @Inject
    DummyDependency dummyDependency;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TextView textView = findViewById(R.id.textView);
        textView.setText(dummyDependency.getApplicationName());
    }
}

File: Tutorial5-1Dagger-Android-ContributesAndroidInjector/src/main/java/com/example/tutorial5_1dagger_android_contributesandroidinjector/MainActivity.kt
package com.example.tutorial5_1dagger_android_contributesandroidinjector

import android.os.Bundle
import android.widget.TextView
import dagger.android.support.DaggerAppCompatActivity
import javax.inject.Inject

class MainActivity : DaggerAppCompatActivity() {

    @Inject
    lateinit var dummyDependency: DummyDependency

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

            findViewById<TextView>(R.id.textView).text = "dummyDependency = $dummyDependency"
    }
}